src/main/drivers/light_ws2811strip.c

```diff
@@ -60,7 +60,11 @@
 #define WS2811_DMA_BUF_CACHE_ALIGN_LENGTH (WS2811_DMA_BUF_CACHE_ALIGN_BYTES / sizeof(uint32_t))
 DMA_RW_AXI __attribute__((aligned(32))) uint32_t ledStripDMABuffer[WS2811_DMA_BUF_CACHE_ALIGN_LENGTH];
 #else
+#ifdef USE_LED_STRIP_FAST_DATA
 FAST_DATA_ZERO_INIT uint32_t ledStripDMABuffer[WS2811_DMA_BUFFER_SIZE];
```

Comment:
It should be something like:
```suggestion
DMA_DATA_ZERO_INIT uint32_t ledStripDMABuffer[WS2811_DMA_BUFFER_SIZE];
```

But DMA_DATA may need some fixing and documentation / new #define for this purpose.

Reply:
> In this state this PR changes behavior of most targets
Really? Imho it reproduces the same behavior present before the #ifdef STM32F7 was removed in c2768d0 

> But DMA_DATA may need some fixing and documentation / new #define for this purpose.
I will look into it, thanks.

Reply:
> Really? Imho it reproduces the same behavior present before the #ifdef STM32F7 was removed in [c2768d0](https://github.com/betaflight/betaflight/commit/c2768d0409a6dd6487bc88a0952079e3048ba43a)

I did not study it closely. But before c2768d0 all F7 targets did use FAST_DATA_ZERO_INIT.  in tis PR, USE_LED_STRIP_FAST_DATA is defined only for two F7 targets

Reply:
Afaik we only have F7X2 and F745 targets - making it dependent on LED_STRIP* is not even needed.

Reply:
https://github.com/betaflight/betaflight/blob/9e53f4ad5b87a762b608c6d6b0f6b765e20c0cae/src/main/target/common_post.h#L76-L82

`USE_FAST_DATA` is defined for 

```
APM32F405xx
APM32F407xx
APM32F415xx
APM32F417xx
STM32F40_41xxx
STM32F7
STM32H7
```

Reply:
@ledvinap this was the previous state

![image](https://github.com/user-attachments/assets/e6e2bdca-90e3-4911-9ada-6b5f66e8a000)

@TheIsotopes please confirm this PR works ?

Reply:
@haslinghuis Yes, I can confirm that this PR works.

Reply:
@ledvinap is the intention to rename `FAST_DATA_ZERO_INIT` to `DMA_DATA_ZERO_INIT` ?

Reply:
USE_LED_STRIP_FAST_DATA makes little sense - ledstrip uses considerable bus bandwidth, so it should use fast memory if possible. It is almost impossible to decipher that it is dma bus/cache related.

Reply:
@ledvinap Yes this PR is a workaround to replace the MCU define removed before.

As this is bug now - we need to fix it. Can you perhaps propose a counter PR ?

Reply:
IMO replacing FAST_DATA_ZERO_INIT with DMA_DATA_ZERO_INIT will fix it, without other changes

Reply:
https://github.com/betaflight/betaflight/blob/f33e1614edd57429657a35f1fc53f379e1c793ba/src/platform/STM32/platform_mcu.h#L217-L221

Reply:
@ledvinap are you sure ?

Reply:
@haslinghuis : 
code before https://github.com/betaflight/betaflight/commit/c2768d0409a6dd6487bc88a0952079e3048ba43a did use FAST_DATA_ZERO_INIT on F7, so DMA_DATA_ZERO_INIT does replicate that on F7.

On f4:
https://github.com/betaflight/betaflight/blob/f33e1614edd57429657a35f1fc53f379e1c793ba/src/platform/STM32/platform_mcu.h#L211-L216, so DMA_DATA_ZERO_INIT is fine

APM32F4 may be broken too; DMA_DATA_ZERO_INIT should fix it.

There is also USE_LED_STRIP_CACHE_MGMT / USE_DSHOT_CACHE_MGMT  hack. All that code should be defined in dma.h, in DMA terms.

`platform/STM32/bus_spi_ll.c:302` seems to handle cache management correctly. `*_CACHE_ALIGN_BYTES` is only complicating code and wasting memory